pages/index.tsx

```diff
@@ -34,32 +37,52 @@ export default function Home({
   errorCode = null,
   userArrivedFromUioMobile = false,
   urlPrefixes,
+  enableGoogleAnalytics,
+  enableMaintenancePage,
 }: HomeProps): ReactElement {
   const { t } = useTranslation('common')
 
+  // Once CSP is enabled, if you change the GA script code, you need to update the hash in csp.js to allow
+  // this script to run. Chrome dev tools will have an error with the correct hash value to use.
+  // https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy/script-src#Unsafe_inline_script
+  const gaScript = `
+        window.dataLayer = window.dataLayer || []
+        function gtag(){dataLayer.push(arguments)}
+        gtag('js', new Date())
+        // For details see: https://support.google.com/analytics/answer/9310895?hl=en
+        // https://developers.google.com/analytics/devguides/collection/gtagjs/ip-anonymization
+        gtag('config', 'UA-3419582-2', { 'anonymize_ip': true }) // www.ca.gov
+        gtag('config', 'UA-3419582-31', { 'anonymize_ip': true }) // edd.ca.gov`
+
+  const googleAnalytics = (
+    <>
+      {/* Global site tag (gtag.js) - Google Analytics */}
+      <script async src="https://www.googletagmanager.com/gtag/js?id=UA-3419582-2" />
+      <script
+        dangerouslySetInnerHTML={{
+          __html: gaScript,
+        }}
+      />
+    </>
+  )
+
   // If any errorCode is provided, render the error page.
-  let mainComponent: JSX.Element
-  if (errorCode) {
-    mainComponent = (
-      <main className="main">
-        <Container className="main-content">
-          <Error statusCode={errorCode} />
-        </Container>
-      </main>
-    )
+  let mainContent: JSX.Element
+  if (enableMaintenancePage) {
+    mainContent = <Maintenance />
+  } else if (errorCode) {
+    mainContent = <Error statusCode={errorCode} />
   } else {
-    mainComponent = (
-      <main className="main">
-        <Container className="main-content">
-          <Title />
-          <ClaimSection
-            loading={loading}
-            userArrivedFromUioMobile={userArrivedFromUioMobile}
-            statusContent={scenarioContent.statusContent}
-            detailsContent={scenarioContent.detailsContent}
-          />
-        </Container>
-      </main>
+    mainContent = (
+      <>
```

Comment:
Is this a typo? (same with the empty closing tag on line 85).

Reply:
it's a react fragment, i'm using it because i can't assign multiple elements otherwise (or as linter says "JSX elements must have one parent element")

https://reactjs.org/docs/fragments.html

Reply:
(i'm anticipating a good emoji reaction)

Reply:
https://reactjs.org/docs/fragments.html#short-syntax 👿 💀